### ISSUE TITLE
Mention List view only conformed to the UITextView

### ISSUE THREAD

**Issue by mrafayfarooq** [linkedin/Hakawai#129]
I want to know if there is a way I can show the list of mentions outside the text view. If my text view is small and has an only a single-line height, my list is hidden because it doesn't have the space to view. Attaching the screenshot for better understanding
![Screen Shot 2019-09-23 at 2 21 03 PM](https://user-images.githubusercontent.com/13344697/65455726-6d3a7700-de0d-11e9-90db-efb6701f3662.png)

**Comment by ashwinshres** [comment]
@mrafayfarooq  did you find any way to show the mention list table outside uitextview???

**Comment by mrafayfarooq** [comment]
> @mrafayfarooq did you find any way to show the mention list table outside uitextview???

No, I realized that you need a table view to show the list. It would be hidden in the beginning but as soon as you enter @ it will be un hidden and populated and will give you the impression that you are using a list

**Comment by iosappsdevelopers** [comment]
Hi @mrafayfarooq 
HI @ashwinshres 

Did you get any solutions for this? can you please share..
